ksqldb-engine/src/main/java/io/confluent/ksql/planner/LogicalPlanner.java

```diff
@@ -663,7 +665,7 @@ private LogicalSchema buildAggregateSchema(
 
   private LogicalSchema buildRepartitionedSchema(
       final PlanNode sourceNode,
-      final Expression partitionBy
+      final List<Expression> partitionBy // TODO: rename?
```

Comment:
i think this is OK

Reply:
Already renamed. Sorry for not squashing my pre-cleanup commits as I usually do. Not worth your time looking at my intermediate thought process while drafting, haha.